modules/mauth-authenticator-akka-http/src/test/scala/com/mdsol/mauth/akka/http/MAuthDirectivesSpec.scala

```diff
@@ -184,4 +196,229 @@ class MAuthDirectivesSpec extends WordSpec with Matchers with ScalatestRouteTest
       }
     }
   }
+
+  "authenticate for V2" should {
```

Comment:
Rename this to be "authenticate when v2OnlyAuthenticate = true" for clarity

Reply:
updated ad suggested https://github.com/mdsol/mauth-jvm-clients/pull/92/commits/735bd0daeb6afd8ec05d1562413a7721f033797f